packages/chart/src/components/Legend.jsx

```diff
@@ -170,13 +174,25 @@ const Legend = () => {
         </LegendOrdinal>
       </aside>
     ) : (
-      <aside id='legend' className={containerClasses.join(' ')} role='region' aria-label='legend' tabIndex={0}>
+      <aside
+        id='legend'
+        className={containerClasses.join(' ')}
+        role='region'
+        aria-label='legend'
+        tabIndex={0} // eslint-disable-line
```

Comment:
fix: remove this line in favor of the error since we should address it for accessibility. Don't mute accessibility issues.

Reply:
Sure I will remove it but whoever you said for me to check with she said just leave it there. i.e. leave tabIndex alone